.travis.yml

```diff
@@ -22,9 +27,12 @@ compiler:
   - gcc
   - clang
 
+python:
+  - '2.7'
+
 env:
   global:
-    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF"
+    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF -DWITH_LIBBOT:BOOL=ON -DWITH_GTK:BOOL=ON -DWITH_DIRECTOR:BOOL=ON -DWITH_IRIS:BOOL=ON-DWITH_OCTOMAP:BOOL=ON -DWITH_MOSEK:BOOL=ON -DWITH_AVL:BOOL=ON -DWITH_XFOIL:BOOL=ON"
```

Comment:
looks like there is a typo?  Missing a space here:

-DWITH_IRIS:BOOL=ON-DWITH_OCTOMAP:BOOL=ON